src/components/views/dialogs/BugReportDialog.tsx

```diff
@@ -65,6 +67,16 @@ export default class BugReportDialog extends React.Component<IProps, IState> {
             downloadProgress: null,
         };
         this.unmounted = false;
+
+        // Get all of the extra info dumped to the console when someone is about
+        // to send debug logs. Since this is a fire and forget action, we do
+        // this when the bug report dialog is opened instead of when we submit
+        // logs because we have no signal to know when all of the various
+        // components have finished logging. Someone could potentially send logs
+        // before we fully dump everything but it's probably unlikely.
+        defaultDispatcher.dispatch({
+            action: Action.DumpDebugLogs,
+        });
```

Comment:
as an optional shortcut:
```suggestion
        defaultDispatcher.fire(Action.DumpDebugLogs);
```

Reply:
I'm going to opt for `dispatch` because it has a lot more usage occurrences to `Ctrl + F` against.